.github/workflows/conformance-aks.yaml

```diff
@@ -139,6 +139,7 @@ jobs:
         uses: ./.github/actions/helm-default
         with:
           image-tag: ${{ inputs.SHA }}
+          chart-dir: ./untrusted/install/kubernetes/cilium
```

Comment:
How does this pull from `untrusted/` if the `untrusted` path is only checked out on line 224?

Reply:
The `helm-default` action [only configures a variable](https://github.com/cilium/cilium/blob/c902a23a8f05162f783abbe870da7ff14e90551a/.github/actions/helm-default/action.yaml#L35), so it doesn't matter if the path has already been checked out at this point, as long as it gets checked out before the actual installation process.